manifests/init.pp

```diff
@@ -335,11 +342,20 @@
     }
   }
 
-  package { $package_name_real:
-    ensure    => $package_ensure,
-    noop      => $package_noop_real,
-    source    => $package_source_real,
-    adminfile => $package_adminfile_real,
+  if $package_manage_real == true {
+    package { $package_name_real:
+      ensure    => $package_ensure,
+      noop      => $package_noop_real,
+      source    => $package_source_real,
+      adminfile => $package_adminfile_real,
+    }
+    $ntp_conf_require      = [ Package[$package_name_real] ]
```

Comment:
thanks for your comment, so could i change the code like below?
  if $package_manage_real == true {
    package { $package_name_real:
      ensure    => $package_ensure,
      noop      => $package_noop_real,
      source    => $package_source_real,
      adminfile => $package_adminfile_real,
    }
  } else {
    package { $package_name_real: }
  }